theme/static/css/hydroshare_core.css

```diff
@@ -1604,10 +1604,6 @@ div#sharing-status {
     overflow: hidden; /* Fixes height for floating elements */
 }
 
-.info-group {
-	width: calc(100% - 220px);
-}
-
```

Comment:
@mauriel133 OK. Got it. Then just make sure you remove all references to this info-group as well, e.g., https://github.com/hydroshare/hydroshare/blob/1075-move-manage-access-button/theme/templates/resource-landing-page/resource-header.html#L4